code/game/atoms.dm

```diff
@@ -619,9 +619,9 @@
   * Produces a signal [COMSIG_PARENT_EXAMINE_MORE]
   */
 /atom/proc/examine_more(mob/user)
-	. = list()
+	. = list("")
 	SEND_SIGNAL(src, COMSIG_PARENT_EXAMINE_MORE, user, .)
-	if(!LAZYLEN(.)) // lol ..length
+	if(. == list("")) // lol ..length
```

Comment:
Heads up--this check will never pass. List equality is checked if the references are the same, not the values.

![image](https://user-images.githubusercontent.com/35135081/96701467-7ca82e80-1345-11eb-831f-7dbea3559ee8.png)

Reply:
I'm not really sure what this is doing here anyway, though.